java/maven.indexer/src/org/netbeans/modules/maven/indexer/OnStop.java

```diff
@@ -41,7 +41,7 @@ public void run() {
                 }
                 if (t.isAlive()) {
                     LOG.warning("...hard stop required.");
-                    t.stop(new Cancellation());
+                    t.interrupt();
```

Comment:
This likely broke the cancellation functionality. (Is `Cancellation` even used now?)

Reply:
In JDK-8 , find below the implemention
 @Deprecated
    public final synchronized void stop(Throwable obj) {
        throw new UnsupportedOperationException();
    }
Unless the method is overridden in some child Thread class, this is least likely to cause an issue